benchmarks/util.py

```diff
@@ -51,28 +49,10 @@ def reset_rng_state(benchmark_experiment=None):
 def is_xla_device_available(devkind):
   if devkind not in ["CPU", "CUDA", "TPU"]:
     raise ValueError(devkind)
-
-  def _check_xla_device(q, devkind):
-    try:
-      import os
-      os.environ["PJRT_DEVICE"] = devkind
-
-      import torch_xla.core.xla_model as xm
-
-      q.put(bool(xm.get_xla_supported_devices(devkind=devkind)))
-    except Exception:
-      traceback.print_exc()
-      q.put(False)
-
-  q = Queue()
-  process = Process(target=_check_xla_device, args=(q, devkind))
-  process.start()
-  process.join(60)
-  try:
-    return q.get_nowait()
-  except queue.Empty:
-    traceback.print_exc()
-    return False
+  CHECK_XLA_DEVICE_PY = "check_xla_device.py"
+  python_file = os.path.join(os.path.dirname(__file__), CHECK_XLA_DEVICE_PY)
+  r = subprocess.run([sys.executable, python_file, devkind])
```

Comment:
Why does this fix the issue? (I never understood why this failure appeared in the first place) 
I just want to make sure that we don't hide the error message in a subprocess and silently fail.